Formula/tio.rb

```diff
@@ -0,0 +1,18 @@
+class Tio < Formula
+  desc "Simple TTY terminal I/O application"
+  homepage "https://tio.github.io"
+  url "https://github.com/tio/tio/releases/download/v1.29/tio-1.29.tar.xz"
+  sha256 "371a11b69dd2e1b1af3ca2a1c570408b1452ae4523fe954d250f04b6b2147d23"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    shell_output("#{bin}/tio -h", 0)
```

Comment:
New formulae require a more substantive test than just help or version -- could you add a test that covers some of `tio`'s functionality?

Reply:
@dirker you could probably use a test similar to that in [epic5.rb](https://github.com/Homebrew/homebrew-core/blob/master/Formula/epic5.rb#L32-L37).

Connect to one of `/dev/tty.***` and kill the process?

Reply:
```rb
test do
  connection = fork do
    exec bin/"tio", "/dev/tty.Bluetooth-Incoming-Port"
  end
  sleep 5
  Process.kill "TERM", connection
end
```

Reply:
I've finally been playing around with your suggestions, thanks! What I found is:
1. tio will error out if passed a pipe rather than a tty on stdin
1. tio will fail to restore the stdin settings if killed via TERM/KILL signals

With regard to point 1, I was trying to send it ctrl-t q in order to ask it to terminate gracefully.

Is point 2 a problem for the tests? When using:
```ruby
test do
  pid = spawn "#{bin}/tio", "/dev/cu.Bluetooth-Incoming-Port"
  sleep 2
  Process.kill "TERM", pid
  Process.wait pid
end
```
my iTerm will end up without echoing characters.